lib/dor/text_extraction/abbyy/results.rb

```diff
@@ -73,6 +76,7 @@ def move_result_files(destination_dir)
         # NOTE: the multiple page Abbyy OCR file is NOT accessioned.
         def paths_to_move
           paths = (output_docs.values + split_ocr_paths).uniq
+          @logger.info("found these candidate paths to move: #{paths}")
           paths.reject { |path| path.end_with?("#{druid}.xml") }
```

Comment:
possibly remove once we verify it all works as expected

Reply:
I've removed it, thanks!